reactor-netty-core/src/main/java/reactor/netty/resources/ConnectionPoolMetrics.java

```diff
@@ -54,4 +54,21 @@
 	 */
 	int pendingAcquireSize();
 
+	/**
+	 * Get the maximum number of live resources this Pool will allow.
+	 * A Pool might be unbounded, in which case this method returns Integer.MAX_VALUE.
+	 * @return the maximum number of live resources that can be allocated by this Pool
```

Comment:
Add `@since 1.0.14`
When a functionality is added after GA, we explicitly indicate in which version this functionality is added.